server.py

```diff
@@ -66,11 +66,17 @@ def process_data(data, connection: socket.socket, user, index):
     elif data[0] == "list":
         return database.list_stocks(user) if not user[3] else database.list_stocks_root()
     elif data[0] == "balance":
+        # Check command
+        if (len(data) != 1):
+            return "400 Invalid command format"
+        
         return database.get_balance(user[0])
     elif data[0] == "deposit":
         # Check command
         if (len(data) != 2):
             return "400 Invalid command format"
+        elif not data[1].isdigit():
```

Comment:
Thanks for catching! Can you add this to the if statement as or since it's the same return

Reply:
Can you add this check to the if statement on line 76 and then I'll go ahead and merge it